views/displaygroup-page.twig

```diff
@@ -57,6 +57,9 @@
 
                             {% set title %}{% trans "Exact match?" %}{% endset %}
                             {{ inline.checkbox("exactTags", title) }}
+
```

Comment:
Can we make this change for 2.3 instead of 2.2? We want to avoid changing functionality in 2.2.3 (as it is a bug fix patch only)